### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by EKaroune**
This is the initial set up of the repository for the Bookdash working group.

Please can this be checked by working group members.

### CODE REVIEW

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+   * Book Dash Chapter and Governance[https://github.com/the-turing-way/the-turing-way/issues/3231#issuecomment-1647578024](https://github.com/the-turing-way/the-turing-way/issues/3231#issuecomment-1647578024)
+   
+
+## Agenda January
+
+   * Officially launch the Working group, identify:
+   * - Chair: Esther
+       * - Chairing meetings
+       * - Liason between the WG and the constitutional levels, communicate decisions etc
+   * - Secretariat (other roles): will be split between Arielle and Emma
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:26]:
```suggestion
   * Secretariat (other roles): will be split between Arielle and Emma
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+       * - ensuring all the topics of the Agenda are covered
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:27]:
```suggestion
       * Ensuring all the topics of the Agenda are covered
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+       * - Notes are documented and uploaded in the repository (still need to decide if this should be an independent repository - we decided yes to the independent repo)
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:28]:
```suggestion
       * Notes are documented and uploaded in the repository (still need to decide if this should be an independent repository - we decided yes to the independent repo)
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+   * - invite other members (invitation link)
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:29]:
```suggestion
   * Invite other members (invitation link)
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+       * - Planning committee is self nominations
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:30]:
```suggestion
       * Planning committee is self nomination from community members who have previously attended a Book Dash
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+       * - WG is previous members of planning committee members as one of the key components
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:31]:
```suggestion
       * WG has previous members of Book Dash planning committee as one of the key components
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+       * - Aiming for 5 people in the WG
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:32]:
```suggestion
       * Aiming for 5 people in the WG
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+       * - List of previous planning committee members. Several of these people are already engaged in WGs and so we don't want to over load them with another WG. We also want more people not Turing staff as EK and AB are turing staff.
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:33]:
```suggestion
       * List of previous planning committee members. Several of these people are already engaged in WGs and so we don't want to overload them with another WG. We also want more people not Turing staff as EK and AB are Turing staff.
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+           * - Susana Roman-Garcia ---> To be asked
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:34]:
```suggestion
           * Susana Roman-Garcia ---> To be asked
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+           * - Carlos - may be too busy  ----> to be asked
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:35]:
```suggestion
           * Carlos - may be too busy  ----> to be asked
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+           * - Andrea Sanchez-Tapia: part of the Translation and localisation
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:36]:
```suggestion
           * Andrea Sanchez-Tapia: part of the Translation and localisation
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+           * - Alden Conner
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:37]:
```suggestion
           * Alden Conner
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+           * - Batool Almarzouq: part of Translation \& Localisation
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:38]:
```suggestion
           * Batool Almarzouq: part of Translation & Localisation WG
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 14]
```diff
@@ -0,0 +1,142 @@
...
+       * - Planning committee is self nominations
+       * - WG is previous members of planning committee members as one of the key components
+       * - Aiming for 5 people in the WG
+       * - List of previous planning committee members. Several of these people are already engaged in WGs and so we don't want to over load them with another WG. We also want more people not Turing staff as EK and AB are turing staff.
+           * - Susana Roman-Garcia ---> To be asked
+           * - Carlos - may be too busy  ----> to be asked
+           * - Andrea Sanchez-Tapia: part of the Translation and localisation
+           * - Alden Conner
+           * - Batool Almarzouq: part of Translation \& Localisation 
+           * - Johanna Bayer
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:39]:
```suggestion
           * Johanna Bayer
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 15]
```diff
@@ -0,0 +1,142 @@
...
+           * - Liz Hare: part of the Accessibility wg
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:40]:
```suggestion
           * Liz Hare: part of the Accessibility WG
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 16]
```diff
@@ -0,0 +1,142 @@
...
+           * - Saranjeet Kaur
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:41]:
```suggestion
           * Saranjeet Kaur
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 17]
```diff
@@ -0,0 +1,142 @@
...
+           * - Winny Nekesa
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:42]:
```suggestion
           * Winny Nekesa
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 18]
```diff
@@ -0,0 +1,142 @@
...
+           * - Alejandro? hasn't been a member of the planning committee
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:43]:
```suggestion
           * Alejandro? Hasn't been a member of the planning committee
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 19]
```diff
@@ -0,0 +1,142 @@
...
+   * - Encourage men to apply to the planning committee
+   * Agree on 2024 Book Dash dates: 
+   * 1st edition: 13-17 May - (TBA: timeline, forms below) / ALE will book rooms at The Turing and add it in the calendar
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:46]:
```suggestion
   * 1st edition: 13-17 May - (TBA: timeline, forms below) / ALE will book rooms at The Turing and add it in the calendar [UPDATE - following a Slack discussion Book Dash is being held 3-6 June instead to accommodate annual leave from team members]
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 20]
```diff
@@ -0,0 +1,142 @@
...
+       * Emma/Ale to set it up
+   * How they can allow others to see their work
+   * Chapter in the Community Handbook
+       * Need to include new information on the WG (next session / Feb)
+   * Use of issue/discussing on main repoREADME, landing page (both in book and .github): 
+
+
+### Timeline / Change dates to May 2024
+
+   * Call for application to join the Book Dash Planning Committee and local hub start date: 15 Feb 2024 (3 months before)
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:81]:
```suggestion
   * Calls for applications to join the Book Dash Planning Committee and host local hubs open: 15 Feb 2024 (3 months before)
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 21]
```diff
@@ -0,0 +1,142 @@
...
+   * Deadline to complete the Expression of Interest to Join *The Turing Way* Book Dash Committee: ~~31 March 2024 ~~1 March 2024 (plus community outreach)
+   * Deadline for local hub - an expression of interest: ~~9 September 2023 ~~1 March 2024
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:83]:
```suggestion
   * Deadline for local hub - an expression of interest: 1 March 2024
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 22]
```diff
@@ -0,0 +1,142 @@
...
+   * How they can allow others to see their work
+   * Chapter in the Community Handbook
+       * Need to include new information on the WG (next session / Feb)
+   * Use of issue/discussing on main repoREADME, landing page (both in book and .github): 
+
+
+### Timeline / Change dates to May 2024
+
+   * Call for application to join the Book Dash Planning Committee and local hub start date: 15 Feb 2024 (3 months before)
+   * Deadline to complete the Expression of Interest to Join *The Turing Way* Book Dash Committee: ~~31 March 2024 ~~1 March 2024 (plus community outreach)
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:82]:
```suggestion
   * Deadline to complete the Expression of Interest to Join *The Turing Way* Book Dash Committee: 1 March 2024 (plus community outreach)
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 23]
```diff
@@ -0,0 +1,142 @@
...
+   * Deadline for local hub - an expression of interest: ~~9 September 2023 ~~1 March 2024
+   * ALE WILL UPDATE THEM / suggestion
+   * Call for Applications for Participants: 24 July 2023
+   * Deadline for submission: 11 September 2023 (anywhere on Earth) - (originally on 15 September 2023 but extended)
+   * Decisions on the applications: Latest by 01 October 2023
+   * Pre Book Dash Onboarding calls (2 x 1 hour): 30 October 2023. 
+       * 9:00 am session: led Batool, Saranjeet, Emma, Anne, Alexandra
+       * 17:00 session: Arielle, Anne, Alexandra
+   * Pre Book Dash GitHub Skill-up (1 hour): May 2024, led by Batool Almarzouq? and Johanna Bayer?
+   * Book Dash Contribution Sessions during the week: of  13-16 May 2024
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:92]:
```suggestion
   * Book Dash Contribution Sessions during the week: of 3-6 June 2024
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 24]
```diff
@@ -0,0 +1,142 @@
...
+   * The Turing Way community share-out: 17 May 2024. Additions to The Turing Way Book
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:93]:
```suggestion
   * The Turing Way community share-out: 7 June 2024. 
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 25]
```diff
@@ -0,0 +1,142 @@
...
+### Timeline / Change dates to May 2024
+
+   * Call for application to join the Book Dash Planning Committee and local hub start date: 15 Feb 2024 (3 months before)
+   * Deadline to complete the Expression of Interest to Join *The Turing Way* Book Dash Committee: ~~31 March 2024 ~~1 March 2024 (plus community outreach)
+   * Deadline for local hub - an expression of interest: ~~9 September 2023 ~~1 March 2024
+   * ALE WILL UPDATE THEM / suggestion
+   * Call for Applications for Participants: 24 July 2023
+   * Deadline for submission: 11 September 2023 (anywhere on Earth) - (originally on 15 September 2023 but extended)
+   * Decisions on the applications: Latest by 01 October 2023
+   * Pre Book Dash Onboarding calls (2 x 1 hour): 30 October 2023.
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:88]:
```suggestion
   * Pre Book Dash Onboarding calls (2 x 1 hour): week beginning 22 May, exact date TBC
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 26]
```diff
@@ -0,0 +1,142 @@
...
+
+### Timeline / Change dates to May 2024
+
+   * Call for application to join the Book Dash Planning Committee and local hub start date: 15 Feb 2024 (3 months before)
+   * Deadline to complete the Expression of Interest to Join *The Turing Way* Book Dash Committee: ~~31 March 2024 ~~1 March 2024 (plus community outreach)
+   * Deadline for local hub - an expression of interest: ~~9 September 2023 ~~1 March 2024
+   * ALE WILL UPDATE THEM / suggestion
+   * Call for Applications for Participants: 24 July 2023
+   * Deadline for submission: 11 September 2023 (anywhere on Earth) - (originally on 15 September 2023 but extended)
+   * Decisions on the applications: Latest by 01 October 2023
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:87]:
```suggestion
   * Decisions on the applications: Latest by 01 May 2024
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 27]
```diff
@@ -0,0 +1,142 @@
...
+
+
+### Timeline / Change dates to May 2024
+
+   * Call for application to join the Book Dash Planning Committee and local hub start date: 15 Feb 2024 (3 months before)
+   * Deadline to complete the Expression of Interest to Join *The Turing Way* Book Dash Committee: ~~31 March 2024 ~~1 March 2024 (plus community outreach)
+   * Deadline for local hub - an expression of interest: ~~9 September 2023 ~~1 March 2024
+   * ALE WILL UPDATE THEM / suggestion
+   * Call for Applications for Participants: 24 July 2023
+   * Deadline for submission: 11 September 2023 (anywhere on Earth) - (originally on 15 September 2023 but extended)
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:86]:
```suggestion
   * Deadline for submission: Monday 15 April (anywhere on Earth) ```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 28]
```diff
@@ -0,0 +1,142 @@
...
+   * Use of issue/discussing on main repoREADME, landing page (both in book and .github): 
+
+
+### Timeline / Change dates to May 2024
+
+   * Call for application to join the Book Dash Planning Committee and local hub start date: 15 Feb 2024 (3 months before)
+   * Deadline to complete the Expression of Interest to Join *The Turing Way* Book Dash Committee: ~~31 March 2024 ~~1 March 2024 (plus community outreach)
+   * Deadline for local hub - an expression of interest: ~~9 September 2023 ~~1 March 2024
+   * ALE WILL UPDATE THEM / suggestion
+   * Call for Applications for Participants: 24 July 2023
```
Comment (Arielle-Bennett) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:85]:
```suggestion
   * Call for Applications for Participants: Tuesday 5 March 2024```

File: README.md [hunk 29]
```diff
@@ -1,2 +1,35 @@
...
+Participants also contribute to enhancing the project by improving the ways we work in the community and take the lead on accomplishing different tasks or sub-projects.
+
+Find out more about the Book Dash [here](https://the-turing-way.netlify.app/community-handbook/bookdash).
+
+![A person helping out another person making their first pull request on GitHub](https://github.com/the-turing-way/bookdash/first-pull-request.svg "First pull request")
+
+## Book Dash Working Group
+
+The Book Dash Working Group was initiated in January 2024. 
+It's purpose is to make long term strategic decisions about the Book Dash events.
```
Comment (EKaroune) [inline README.md:15]:
```suggestion
It's purpose is to make long term strategic decisions about the Book Dash events - to be decided and agreed in a future meeting.
```

File: README.md [hunk 30]
```diff
@@ -1,2 +1,35 @@
...
+* Arielle Bennett - Secretary
+* Emma Karoune - Secretary
+* Carlos Martinez
+* Susana Roman Garcia
+
+Our working group meeting notes can be found [here](https://github.com/the-turing-way/bookdash/wg-meeting-notes)
+
+## Book Dash Planning Committee
+Current members of our community that have previously joined a Book Dash can apply to be on the planning committee for an individual Book Dash event.
+
```
Comment (EKaroune) [inline README.md:30]: At least one Book Dash Working Group member will attend planning committee meetings to support communication between the two groups.

File: README.md [hunk 31]
```diff
@@ -1,2 +1,35 @@
...
+* Susana Roman Garcia
+
+Our working group meeting notes can be found [here](https://github.com/the-turing-way/bookdash/wg-meeting-notes)
+
+## Book Dash Planning Committee
+Current members of our community that have previously joined a Book Dash can apply to be on the planning committee for an individual Book Dash event.
+At least one Book Dash Working Group member will attend planning committee meetings to support communication between the two groups.
+Find out more about the Book Dash planning committee [here](https://the-turing-way.netlify.app/community-handbook/bookdash/bookdash-selection).
+
+
```
Comment (EKaroune) [inline README.md:33]:
```suggestion
Need to discuss more in next meeting how we want the two groups to interact.
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 32]
```diff
@@ -0,0 +1,142 @@
...
+## Date: 31 January 2024
+
+
+## About this call
+
+The Turing Way Book Dash Working Group meets on a monthly basis 
+
+### Code of conduct
+
+\_The Turing Way\_ Code of conduct applies to this call. Take a moment to read this: [https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE\_OF\_CONDUCT.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE\_OF\_CONDUCT.md)
```
Comment (EKaroune) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:11]:
```suggestion
*The Turing Way* Code of conduct applies to this call. Take a moment to read this: [https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE\_OF\_CONDUCT.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE\_OF\_CONDUCT.md)
```

File: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md [hunk 33]
```diff
@@ -0,0 +1,142 @@
...
+    - Saranjeet Kaur
+
+    - Winny Nekesa
+
+###  Decisions / Action Items
+
+   * Invite Susana and Carlos to join WG in first instance - Esther (done)
+   * There is a need for a more balanced ratio of genders in the Book Dash planning committee
+   * Community outreach to previous planning committee members and specific Book Dash attendees for the next planning committee applications - to be divided up 
+   * First Book Dash of 2024 will be 13-17 May 2024
```
Comment (AlexandraAAJ) [inline wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md:133]: After some Slacks discussions, the agreed dates for the first Book Dash are 3-7 June 2024

**Review by AlexandraAAJ — COMMENTED** [review]
Apart form one comment to update the Book Dash dates, it looks perfect to me.

**Pull request merged**